packages/gestalt/src/ButtonSocial.tsx

```diff
@@ -0,0 +1,325 @@
+import { forwardRef, useImperativeHandle, useRef } from 'react';
+import getAriaLabel from './accessibility/getAriaLabel';
+import NewTabAccessibilityLabel from './accessibility/NewTabAccessibilityLabel';
+import Box from './Box';
+import VRButtonSocial from './ButtonSocial/VRButtonSocial';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider';
+import Flex from './Flex';
+import Icon from './Icon';
+import InternalLink from './Link/InternalLink';
+import Text from './Text';
+import useInExperiment from './useInExperiment';
+
+const TEXT_OPTIONS = {
+  1: 'Login with',
+  2: 'Continue with',
+  3: 'Sign up with',
+};
+
+const SERVICES_OPTIONS = {
+  1: 'Apple',
+  2: 'Facebook',
+  3: 'Google',
+  4: 'Email',
+};
+
+type ButtonProps = {
+  /**
+   * Label to provide more context around ButtonLink’s function or purpose. See the [Accessibility guidelines](/foundations/accessibility) to learn more.,
+   */
+  accessibilityLabel?: string;
+  /**
+   * Available for testing purposes, if needed. Consider [better queries](https://testing-library.com/docs/queries/about/#priority) before using this prop.
+   */
+  dataTestId?: string;
+  /**
+   * Indicates whether this component is hosted in a light or dark container.
+   * Used for improving focus ring color contrast.
+   */
+  focusColor?: 'lightBackground' | 'darkBackground';
+  /**
+   * Default Buttons are sized by the text within the ButtonSocial whereas full-width Buttons expand to the full width of their container.
+   */
+  fullWidth?: boolean;
+  /**
+   * Use "-1" to remove ButtonSocial from keyboard navigation. See the [Accessibility guidelines](/foundations/accessibility) to learn more.
+   */
+  tabIndex?: -1 | 0;
+  /**
+     * Callback invoked when the user clicks (press and release) on ButtonSocial with the mouse or keyboard.
+       See [GlobalEventsHandlerProvider](/web/utilities/globaleventshandlerprovider#Link-handlers) to learn more about link navigation.
+     */
+  onClick?: (arg1: {
+    event: React.MouseEvent<HTMLAnchorElement> | React.KeyboardEvent<HTMLAnchorElement>;
+    dangerouslyDisableOnNavigation: () => void;
+  }) => void;
+  /**
+   * Text to render inside the ButtonSocial to convey the function and purpose of the ButtonSocial.
+   */
+  text: keyof typeof TEXT_OPTIONS;
+  /**
+   * Text to render inside the ButtonSocial to convey the function and purpose of the ButtonSocial.
+   */
+  service: keyof typeof SERVICES_OPTIONS;
+  /**
+   * Specifies a link URL.
+   */
+  href: string;
+  /**
+   * Provides hints for SEO. See the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#rel) to learn more.
```

Comment:
Let's just copy paste whatever the Link component has:
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/a5a53a0c-7f4f-4333-ae29-3c606759f7c7)



Establishes the relationship of the linked URL. Use rel="nofollow" for offsite links to inform search engines to ignore and not follow them. See the externalLinkIcon and rel variant to learn more.

https://deploy-preview-3994--gestalt.netlify.app/web/link